### PR TITLE
[Javascript] Add nanoexpress Pro

### PR DESCRIPTION
**Pull request by dalisoft**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] A `Dockerfile` exists, or can be built ?
+ [ ] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [x] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [x] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

### CODE REVIEW

File: javascript/nanoexpress-pro/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+framework:
+  github: dalisoft/nanoexpress#pro
+  website: nanoexpress.js.org
```
Comment (waghanza) [inline javascript/nanoexpress-pro/config.yaml]: could you use only one markup, rather `website`

File: javascript/nanoexpress-pro/config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  version: 1.2.8
```
Comment (waghanza) [inline javascript/nanoexpress-pro/config.yaml]: could you use X.Y format ?

File: javascript/nanoexpress-pro/package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress-pro": "^1.2.8"
```
Comment (waghanza) [inline javascript/nanoexpress-pro/package.json]: could you use `~1.2.8` to keep in `1.2` ?

File: javascript/nanoexpress-pro/config.yaml (hunk 1 above)
Comment (dalisoft) [reply javascript/nanoexpress-pro/config.yaml]: Ok, i will fix it

File: javascript/nanoexpress-pro/config.yaml (hunk 2 above)
Comment (dalisoft) [reply javascript/nanoexpress-pro/config.yaml]: Yes

File: javascript/nanoexpress-pro/package.json (hunk 3 above)
Comment (dalisoft) [reply javascript/nanoexpress-pro/package.json]: But i want upgrade to latest version as possible when i change happens in versions

Comment (waghanza) [reply javascript/nanoexpress-pro/package.json]: I understand, but to display the version in results, I need a version range (in a **minor**).

There is a @dependabot that create `PR` for updates

Comment (dalisoft) [reply javascript/nanoexpress-pro/package.json]: Ok, i fix it :)

**Pull request merged**